modules/rosa-install-logging-addon.adoc

```diff
@@ -0,0 +1,76 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_logging/rosa-install-logging.adoc
+
+[id="rosa-install-logging-addon_{context}"]
+= Install logging add-on services on {product-title}
+
+{product-tite} (ROSA) does not come preconfigured with a logging solution, but logging can be enabled by manually installing the `cluster-logging-operator` service.
+
+Log forwarding capability is provided through the Amazon CloudWatch add-on service.
+
+.Prerequisites
+
+Request the following link:https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html[quotas from AWS]:
+* Quota for the `cluster-logging-operator` add-on
+* Quota for the Amazon CloudWatch add-on
+
+.Procedure
+
+To install log forwarding using interactive mode:
```

Comment:
Ok, but that's not great for 508 compliance.

Reply:
I can't speak to that, but it's part of our standards, so that's what we should follow. I do think that accessibility was brought up as a consideration when this item was being discussed/added to the standards.